core-bundle/src/EventListener/InsertTags/DateListener.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\InsertTags;
+
+use Contao\Config;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\Hook;
+use Contao\Date;
+use Contao\PageModel;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Terminal42\ServiceAnnotationBundle\ServiceAnnotationInterface;
+
+/**
+ * Provides a {{format_date::<timestamp>::<format>}} and a
+ * {{convert_date::<date>::<source_format>::<target_format>}} Insert Tag.
```

Comment:
```suggestion
 * {{convert_date::<date>::<source_format>::<target_format>}} insert tag.
```

Because "insert tag" is not a product name.